src/Symfony/Component/Translation/Resources/functions/translatable.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\Component\Translation\Translatable;
+
+if (!function_exists('_t')) {
+    /**
+     * @author Nate Wiebe <nate@northern.co>
+     */
+    function _t(string $messageId, array $parameters = [], string $domain = 'messages'): Translatable
```

Comment:
`t`?

Reply:
I personally prefer having the underscore as it reduces function collisions since we have 2 chars instead of 1 (which I guess shouldn't necessarily be our concern). But also looking at some of the major php communities, it appears to be common practice to use an underscore to prefix translation related functions. We don't necessarily need to do what others are doing, but I think it would help people transition between different projects that might be using different frameworks.

Underscore:
- `_()` - https://www.php.net/_
- `__()` - https://devdocs.magento.com/guides/v2.4/frontend-dev-guide/translations/translate_theory.html
- `__()` - https://laravel.com/docs/7.x/localization
- `_e()` - https://developer.wordpress.org/reference/functions/_e/

Non-underscore:
- `t()` - https://api.drupal.org/api/drupal/core%21includes%21bootstrap.inc/function/t/9.0.x

Reply:
Just as a note, I believe Drupal uses Symfony as a base, so having `t()` might conflict with Drupal if they are using the translation component.

Reply:
it's a namespaced function :) to me there's no real reason to have a prefix

i like _() also, but t() seems a bit more explicit (or "less weird" :))

Reply:
Okay, this will also match up with `b()`, `s()`, and `u()`

Reply:
Updated